test/evergreen.yml

```diff
@@ -272,9 +274,9 @@ functions:
           # Compiling with CMake generated Ninja file.
           cd cmake_build
           if [ "${cmake_generator|Ninja}" = "Ninja" ]; then
-            ninja ${smp_command|} 2>&1
+            ninja ${num_parallel} 2>&1
```

Comment:
As discussed, I think this is confusing as from the variable name, you don't expect the `-j` to be included. The `smp_command` was probably a good name for that reason.